tests/application/use_cases/test_auth.py

```diff
@@ -49,10 +49,19 @@ def test_get_or_create_user_invalid_email(app_context):
 
 
 def test_get_logged_in_user_redirect_url(app_context):
-    app_context.auth.make_login_url = lambda provider_name, state_token: "test"
+    app_context.auth.make_login_url = (
+        lambda provider_name, state_token, acr_values=None, nonce=None: "test"
+    )
+
     provider_name = "test"
     state_token = "test"
```

Comment:
For clarity, I'd suggest giving these unique names, so there's more confidence that the assertion at the end of this test is referring to the URL and not one of these values.